src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalController.java

```diff
@@ -638,4 +639,39 @@ public ResponseEntity<Object> updateAnOrganisationsRegisteredPbas(
                 .status(updatePbaStatusResponse.getStatusCode())
                 .body(updatePbaStatusResponse);
     }
+
+    @Operation(
+            summary = "Retrieves the organisation details of a user",
+            description = "**IDAM Roles to access API** : <br> prd-admin",
+            security = {
+                    @SecurityRequirement(name = "ServiceAuthorization"),
+                    @SecurityRequirement(name = "Authorization")
+            }
+    )
+
+    @ApiResponse(
+            responseCode = "200",
+            description = "",
+            content = @Content(schema = @Schema(implementation = OrganisationEntityResponse.class))
+    )
+    @ApiResponse(
+            responseCode = "403",
+            description = "Forbidden Error: Access denied",
+            content = @Content
+    )
+    @ApiResponse(
+            responseCode = "500",
+            description = "Internal Server Error",
+            content = @Content

```

Comment:
@kiran-yenigala-hmcts , missing status code 400 and 404 in rest api doc, pls add.

Reply:
done